pype/plugins/global/publish/integrate_master_version.py

```diff
@@ -513,75 +510,6 @@ def copy_file(self, src_path, dst_path):
 
         shutil.copy(src_path, dst_path)
```

Comment:
this could make use of **speedcopy**

Reply:
Not part of this PR. Create new issue please.